src/applications/disability-benefits/all-claims/utils/form0781.js

```diff
@@ -40,6 +40,14 @@ export function showForm0781Pages(formData) {
   );
 }
 
+export function showManualUpload0781Page(formData) {
```

Comment:
I didn't see a test for this function.

Reply:
Yeah remember we discussed these utility functions and agreed anything relating to the page routing was best saved for e2e tests? This one ended up being a [real rabbit hole](https://dsva.slack.com/archives/C053U7BUT27/p1734729884214569) because of the need to do some stubbing our test library doesn't easily support. I spent a few days trying to get that to work and stopped because we discussed it and decided we could cover it later with e2e tests.